### PR TITLE
Accessory show/kelly

### PR DESCRIPTION
**Pull request by anubiskhan**
Changes proposed in this pull request:  
- Adds accessories table w/ status column to check retired or not
- Adds MVC components for accessory
- Adds accessories#show view and route
- Adds all requisite tests

connect to #14 connect to #15 
@adam-conway @anubiskhan @vadlusk @agpiermarini

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -15,6 +15,16 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "accessories", force: :cascade do |t|
+    t.string "title"
+    t.text "description"
+    t.decimal "price", precision: 9, scale: 2
```
Comment (adam-conway) [inline db/schema.rb]: What does the precision and scale do in this context?

Comment (anubiskhan) [reply db/schema.rb]: The precision denotes how many characters the number can be, the scale denotes how many characters can be on the right of the decimal point. From what I found online it is the best way to setup your db to handle a column that will be a currency.

**Comment by anubiskhan** [comment]
Also, skipped test in accessory show because the link needs to interact with cart.

**Comment by adam-conway** [comment]
Looks good!

**Pull request merged**